conformance/ion_literal.ion

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+(ion_1_1
+  (each (toplevel '#$11'             '#$11'::'#$11'             '#$11'::1)
+        (toplevel $ion_literal $ion_literal::$ion_literal $ion_literal::1)
+        (produces $ion_literal               $ion_literal               1)))
```

Comment:
What about `$ion_literal::$ion_literal::1`—would that "expand" to `$ion_literal::1`?

Reply:
I added that to the test cases.